framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1281,6 +1309,21 @@ private VariableElement findElement(TypeElement typeElt, VariableDeclarator vari
         return findFieldElement(typeElt, fieldName);
     }
 
+    /**
+     * Looks for an enum constant element in the typeElt and returns it if the element has the same
+     * name as provided. In case enum constant element is not found it returns null.
+     *
+     * @param typeElt type element where enum constant element should be looked for
+     * @param enumConstDecl the declaration of the enum constant
+     * @return enum constant element in typeElt with the provided name or null if enum constant
+     *     element is not found
+     */
+    private VariableElement findElement(
```

Comment:
How about moving this a bit further up to be next to `private Element findElement(TypeElement typeElt, EnumDeclaration enumDecl)` and keeping the related methods together?